src/matlab2tikz.m

```diff
@@ -5624,29 +5634,23 @@ function printAll(m2t, env, fid)
     versionString = '';
 end
 % ==============================================================================
-function isHG2 = isHG2(m2t)
+function tf = isHG2(m2t)
 % Checks if graphics system is HG2 (true) or HG1 (false).
 % HG1 : MATLAB up to R2014a and currently all OCTAVE versions
 % HG2 : MATLAB starting from R2014b (version 8.4)
-    isHG2 = false;
-    if strcmpi(m2t.env,'MATLAB') && ...
-        ~isVersionBelow(m2t.env, m2t.envVersion, [8,4])
-        isHG2 = true;
-    end
+    tf = strcmpi(m2t.env,'MATLAB') && ...
+        ~isVersionBelow(m2t.env, m2t.envVersion, [8,4]);
 end
 % ==============================================================================
-function isBelow = isVersionBelow(env, versionA, versionB)
+function tf = isVersionBelow(env, versionA, versionB)
```

Comment:
@okomarov My personal convention is to use `bool` (or a name that actually means something like `isValid`).

I know at TMW, most of the time `tf` is used as boolean return value, but I prefer to not use it, because it shadows the `tf`  function/object (i.e. a transfer function parametrized as polynomials from their own Control Toolbox, which I use a lot for my work) and it may cause confusion in code where both are used at the same time (not likely to happen in `m2t` though).
The other reason why I dislike `tf`, is because it doesn't really mean anything except `true-false` in an obfuscated form. I also find the term `tf` for `truefalse` a bit immature: you wouldn't call an enum after all possible values it can take (e.g. "SmallMediumLarge"), but rather something meaningful, such as "shirtSize".

On the other hand, `bool` (short for boolean) is something that is present in some other programming languages, so my assumption is that such a name is more familiar to a wider audience. But it actually conveys the same info as `tf`: this value is supposed to be boolean value. But to be honest:  `bool` is just my own lazy alternative to `tf` for when I don't want to think about a proper name.

MATLAB handles boolean variables in quite a few dirty ways:
*  actual `logical` (i.e. `true` vs `false`) variables (i.e. booleans in most other languages),
* `'on'` vs `'off'` "booleans" (actually strings) that are still used in the HG framework and some other places,
* implicit conversions, e.g. the output of `exist()` that we talked about a while ago.
So I would argue that TMW isn't always the best example to follow.

But of course, this is a matter of taste. Personally, I prefer `isValid` since that actually means something and that is also the convention in many other languages.
If I'm not mistaken, that's also what most MATLAB style guides recommend.

But if there's a consensus of everybody else to use something specific, also fine by me.